models/index.js

```diff
@@ -2,17 +2,16 @@
 
 const fs = require('fs');
 const path = require('path');
-const Sequelize = require('sequelize');
-const basename = path.basename(__filename);
+var Sequelize = require('sequelize');
```

Comment:
not sure if we need this change just yet ? @mmoncayo @reptile18 do you have feedback?